crates/hyperqueue/src/client/commands/job.rs

```diff
@@ -129,11 +121,14 @@ pub async fn output_job_list(
 pub async fn output_job_detail(
     gsettings: &GlobalSettings,
     connection: &mut ClientConnection,
-    selector: Selector,
+    selector: IdSelector,
 ) -> anyhow::Result<()> {
     let message = FromClientMessage::JobDetail(JobDetailRequest {
-        selector,
-        include_tasks: true,
+        job_id_selector: selector,
+        task_selector: Some(TaskSelector {
+            id_selector: IdSelector::All,
+            status_selector: StatusSelector::All,
```

Comment:
The CLI printer currently only cares about failed jobs, but in JSON all tasks are printed. Maybe later we could "optimize" this by only downloading failed tasks if the CLI printer is used. But don't do that in this PR, it's not a problem currently.